be/src/storage/CMakeLists.txt

```diff
@@ -210,6 +210,20 @@ add_library(Storage STATIC
     lake/vertical_compaction_task.cpp
     lake/metacache.cpp
     lake/lake_primary_key_recover.cpp
+    sstable/block_builder.cpp
```

Comment:
wondering if it worths a separate .a archive.

Reply:
I only want to use sstable format and do some hack on it. I don't think we can use it as a lib.